### PR TITLE
Suspended now counts as failed

### PR DESCRIPTION
**Pull request by efarka**

### CODE REVIEW

File: final project cleaning.Rmd [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+
+```{r}
+projects <- read.csv("C:/Users/efarka/Downloads/kickstarter-projects/ks-projects-201801.csv")
+
+
+#to time series
+projects$launched <- strptime(projects$launched, "%Y-%m-%d %H:%M:%S")
+projects$deadline <- strptime(projects$deadline, "%Y-%m-%d")
```
Comment (segevm) [inline final project cleaning.Rmd:13]: Don't we want both these variables to be in the same date format for consistency in calculations? I feel like we should just remove the %H:%M:%S from projects$launched

File: final project cleaning.Rmd [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#make duration column
+projects$duration = difftime(projects$deadline, projects$launched, units="days")
+
+
+#only care about these three states
+projects$state2[projects$state=="successful"] <- 1
+projects$state2[projects$state=="failed"] <- 0
+projects$state2[projects$state=="canceled"] <- 0
+projects$state2[projects$state=="suspended"] <- 0
```
Comment (segevm) [inline final project cleaning.Rmd:22]: Missing the "live" and "undefined" states

Also, don't we want to factor the projects$state2?

**Review by segevm — CHANGES_REQUESTED** [review]

File: final project cleaning.Rmd (hunk 1 above)
Comment (efarka) [reply final project cleaning.Rmd:13]: Sure!

File: final project cleaning.Rmd (hunk 2 above)
Comment (efarka) [reply final project cleaning.Rmd:22]: Live and undefined are removed in the line below this.  They don't really fit cleanly into successful or failed so it makes more sense to not analyze them.  
I kinda coded it in a weird way lol but the state2 ends up replacing the state column later on. I drop the original state column and rename state2 to state.

**Pull request closed without merging**